single.html

```diff
@@ -3,65 +3,140 @@
 
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Example tea</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Acme&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+  <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/single_style.css">
+  <title>Greenfield black tea</title>
 </head>
 
 <body>
 
   <header>
     <div class="container">
-      <a href="#" class="header__logo">
-        <img src="img/logo220.png" alt="">
+      <a href="index.html" class="header__logo">
+        <img src="img/logo_desctop.png" alt="">
       </a>
       <div class="header__bottom">
+        <div class="header__burger"><span></span></div>
         <nav>          
-          <a href="#" class="header__button">Green tea</a>
-          <a href="#" class="header__button">Black tea</a>
-          <a href="#" class="header__button">Other</a>
-          <input type="search" placeholder="Search">
+          <a href="index.html" class="header__button" data-text="Home">Home</a>
+          <a href="gallery.html" class="header__button" data-text="Gallery">Gallery</a>
+          <a href="add_tea.html" class="header__button" data-text="Home">Add tea</a>
         </nav>
-        <div class="header__user__menu">
-          <a href="#" class="header__button">Log In</a>
-          <a href="#" class="login__logo">
-              <img src="img/login.png" alt="login">
-          </a>
+        <div class="header__login">
+          <a href="#" class="header__button" id="login">Log In</a>
```

Comment:
I don't think it should be `a`.

Reply:
```suggestion
          <button class="header__button" id="login">Log In</button>
```
Truly, changed it to button

Reply:
done